libff/algebra/curves/bls12_377/bls12_377_init.cpp

```diff
@@ -341,17 +528,37 @@ void init_bls12_377_params()
     // Pairing parameters
     // sage: u = 9586122913090633729
     // sage: ceil(log(u, 2)) # = 64
-    // sage: bin(u) # = '0b1000010100001000110000000000000000000000000000000000000000000001'
-    // The Hamming weight of u is: HW(u) = 7
-    // u = 2**63 + 2**58 + 2**56 + 2**51 + 2**47 + 2**46 + 1
-    // Based on the power-2 decomposition of u, we should have 63 doubling steps and 7 addition steps in the Miller Loop.
+    // sage: bin(u) # =
+    // '0b1000010100001000110000000000000000000000000000000000000000000001' The
+    // Hamming weight of u is: HW(u) = 7 u = 2**63 + 2**58 + 2**56 + 2**51 +
```

Comment:
@dtebbs do you know if we can enforce some format on the comments? I find the comments after clang-format to be less readable here.

Reply:
Tidied this up in the latest.
In general, it seems clang-format will not join lines together if there are indents and/or line spaces (which often also helps with readability fo the comments).

Reply:
I see, good to know. Thanks for fixing the formatting @dtebbs